### PR TITLE
Update utilities with upstream changes

### PR DESCRIPTION
**Pull request by b6b**
We need `netsuite_data_center_urls` to get account-based data center urls.

### CODE REVIEW

**Review by arincon9 — APPROVED** [review]
LGTM 🥇

File: Gemfile [hunk 1]
```diff
@@ -6,6 +6,10 @@ gem 'simplecov', :require => false
 gem 'pry-nav'
 gem 'pry-rescue'
 
+# optional dependency for more accurate timezone conversion
+gem 'tzinfo', '1.2.5'
+# gem 'tzinfo', '2.0.0'
```
Comment (kcotugno) [inline Gemfile:11]: Shouldn't there be changes to the lock file?
Can we use the newer version or at least remove the commented out version?

Comment (b6b) [reply Gemfile:11]: `Gemfile.lock` is actually in the gitignore. For some reason upstream they commented out `2.0.0` to go back to `1.2.5` so I just left it as-is.

Comment (kcotugno) [reply Gemfile:11]: Sounds good.

**Pull request merged**